lib/modules/datasource/unity3d/index.spec.ts

```diff
@@ -23,12 +24,38 @@ describe('modules/datasource/unity3d/index', () => {
     }
   };
 
+  const createUnityReleases = (
+    total: number,
+    offset: number,
+    resultCount: number,
+  ) {
+    const results = [];
+
+    for (let i = 0; i < resultCount; i++) {
+      results.push({
+        version: `6000.0.${offset + i + 1}f1`,
```

Comment:
If you loop from 1 up to and including the `resultCount`, you don't need to increment every single loop:
```suggestion
    for (let i = 1; i <= resultCount; i++) {
      results.push({
        version: `6000.0.${offset + i}f1`,
```